stable/aws-global-accelerator-controller/values.yaml

```diff
@@ -53,7 +53,14 @@ affinity: {}
 
 webhook:
   enabled: true
-  certificateName: aws-global-accelerator-cert
+  port: 8443
   serviceName: aws-global-accelerator-validation
-  issuerName: issuer
-  certificateSecretName: aws-global-accelerator-tls
+  https:
+    enabled: true
```

Comment:
Why do you prepare two parameters?
- `https.enabled`
- `certManager.enabled`

Are you supposed to use `https` with disabled and `certManager` with enabled or `https` with enabled` and `certManager` with disabled?

Reply:
You could use https without certmanager, but it wouldn't be possible to use certmanager without https. I could nest certmanager inside of the https option?

Reply:
Ah, sorry, you're right. If we specify certain certificates, we can use https without cert-manager.